main.py

```
@@ -41,6 +43,40 @@ def __init__(self, owner: str, repo: str, plugins_dir: Path):
         self._repo = repo
         self._plugins_dir = plugins_dir
 
+    def _install_deps(self, plugin_dir: Path) -> str:
+        """Install plugin requirements into the embedded Python's site-packages.
+
+        Uses sys.executable (the embedded runtime\python.exe in a release build,
```

Ruff (0.15.10)

[warning] Invalid escape sequence: `\p`

Use a raw string literal

(W605)